docs/docsite/rst/user_guide/playbooks_filters.rst

```diff
@@ -875,6 +875,10 @@ You can initialize the random number generator from a seed to create random-but-
 
     "{{ 60 | random(seed=inventory_hostname) }} * * * * root /script/from/cron"
 
+.. note::
+
+  While using ``seed`` parameter, the resulting value can be different depending on the version of Python used on the Ansible controller.
```

Comment:
```suggestion
  If you use the ``seed`` parameter, you will get a different result with Python 3 and Python 2. This may break procedures like password generation when you upgrade the version of Python used on your Ansible controller.
```